packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "3f504ec18ba5925efbc7245c871cc0583b2f7890",
-    "ref_origin" : "dcos-mesos-1.2.0"
+    "ref": "6adb3a3071524b87d7ce97d371e82aa2a7af6cbd",
+    "ref_origin" : "dcos-mesos-1.2.x-801cdfac"
```

Comment:
Could be a little closer to today's 1.2.x HEAD, but we'll probably bump it again once @tillt 's env changes land.